BrentOzarULTD/SQL-Server-First-Responder-Kit
# Add checks for implicit transactions

Fixes #1459

Changes proposed in this pull request:
 - Adds checks for implicit transactions to Blitz, BlitzWho, and BlitzFirst
 - While working on this, I found some mis-numbered CheckIds in Blitz and corrected those
 - Took the opportunity to give BlitzFirst a semicolonoscopy. Used SQL Prompt. No manual labor here.

How to test this code:
 - See relevant demos in the issue - #1459

Has been tested on (remove any that don't apply):
 - Case-sensitive SQL Server instance
 - SQL Server 2016
  - SQL Server 2017
